pkg/vz/vz_driver_darwin.go

```diff
@@ -191,8 +191,7 @@ func (l *LimaVzDriver) RunGUI() error {
 	if l.CanRunGUI() {
 		return l.machine.StartGraphicApplication(1920, 1200)
 	}
-	//nolint:revive // error-strings
-	return fmt.Errorf("RunGUI is not supported for the given driver '%s' and display '%s'", "vz", *l.Instance.Config.Video.Display)
+	return fmt.Errorf("runGUI is not supported for the given driver '%s' and display '%s'", "vz", *l.Instance.Config.Video.Display)
```

Comment:
The string `runGUI` does not make any sense to the user; maybe we should list the option name the user specified instead?

```suggestion
	return fmt.Errorf("--run-gui is not supported for the given driver '%s' and display '%s'", "vz", *l.Instance.Config.Video.Display)
```

Reply:
Might be confusing, as the --run-gui flag is not visible to limactl user

Reply:
Reverted to `RunGUI`, should be fine.